charts/rancher-monitoring/rancher-monitoring/100.1.0+up19.0.3/charts/grafana/templates/podsecuritypolicy.yaml

```diff
@@ -13,7 +13,18 @@ spec:
   allowPrivilegeEscalation: false
   requiredDropCapabilities:
     # Default set from Docker, with DAC_OVERRIDE and CHOWN
-      - ALL
+    - FOWNER
```

Comment:
What is not dropped by this? does that just leave chown?

Reply:
[K8s docs on this](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#capabilities) redirect you to [this man page](https://man7.org/linux/man-pages/man7/capabilities.7.html). There's a substantial deal of other capabilities that will no longer be dropped.

Reply:
Good question, I just copied the original list from the upstream and did not check if the list covers all. 
I think what I can do is to add all except ` CHOWN` to the list to see if it works.

Reply:
This being said, [docker](https://docs.docker.com/engine/security/) does restrict capabilities to a [limited list](https://github.com/moby/moby/blob/master/oci/caps/defaults.go#L6-L19) and this requires that they all be dropped besides CHOWN and DAC_OVERRIDE. But docker is not the only container engine so this might be a valid concern. I'll remove my approval until we resolve this discussion.

Reply:
I updated the PR to include the `DAC_OVERRIDE` to the list of  `requiredDropCapabilities`. As the result, `CHOWN` is the only permission that is allowed by the PSP.
I do not know why `DAC_OVERRIDE` is preserved in the original chart, but I tested the modified chart in a hardened cluster and can confirm Grafana works properly without `DAC_OVERRIDE`. 

By default, when `AllowedCapabilities` is not set/empty, it means no additional capabilities may be added beyond the default set (reference). I confirmed that [k8s' default capability set](https://kubernetes.io/docs/concepts/security/pod-security-standards/#:~:text=Capabilities,must%20be%20disallowed.) is a subset of [Docker's capability default ](https://docs.docker.com/engine/reference/run/#:~:text=can%20be%20dropped.-,Capability%20Key,Capability%20Description,-AUDIT_WRITE). (`NET_RAW` is in docker's not in k8s').  

As to docker is not the only container engine, since Docker is the only officially supported container engine for Rancher, we only need to confirm the chart working properly when Docker is used.